test/test_vmap.py

```diff
@@ -3008,7 +3036,6 @@ def test_vmap_exhaustive(self, device, dtype, op):
         xfail('nn.functional.cross_entropy', 'mean'),
         xfail('nn.functional.cross_entropy', 'none'),
         xfail('nn.functional.cross_entropy', 'sum'),
-        xfail('nn.functional.grid_sample'),
```

Comment:
I took this skip out but left in the skip in [text_ops.py](https://github.com/facebookresearch/functorch/blob/grid_sample_registration/test/test_ops.py#L444) because that test skips linear and conv2d later down. Not sure if there's something else I need to do to make that test work, but wanted to call it out

Reply:
That test skips linear and conv2d probably because they break :P Specifically for conv, it probably looks for a batching rule for conv_backward as well, which I don't think we have right now.